setup.py

```diff
@@ -149,179 +134,23 @@ def append(self, item):
-        compiler = self._get_compiler()
-        extra_postargs = ["-std=c99", "-pedantic",
-                          "-Wextra", "-Wno-missing-field-initializers",
-                          "-DH5_NO_DEPRECATED_SYMBOLS"]
-
-        # Compile and link any sources that are passed in
-        output_dir = os.path.join(self.build_dir, arch)
-        try:
-            binaries = []
-            for bin, srcs in self.scripts.items():
-                # Only compile srcs with c files
-                try:
-                    if not any([src.endswith(".c") for src in srcs]):
-                        continue
-                except AttributeError:
-                    if not src.endswith(".c"):
-                        continue
-
-                objs = compiler.compile(srcs, output_dir=output_dir,
-                                        include_dirs=include_dirnames,
-                                        extra_postargs=extra_postargs,
-                                        debug=False)
-
-                bin_path = os.path.join(output_dir, bin)
-
-                compiler.link_executable(objs, bin_path,
-                                         library_dirs=library_dirnames)
-                binaries.append(bin_path)
-
-            # Replace dict script with actual before build_scripts.run() call
-            self.scripts = binaries
-        except AttributeError:  # Not a dict
-            pass
-
-        print("##################################################")
-
-        build_scripts.run(self)  # Call actual script
-
-        # If success, remove script build dir
-        if os.path.isdir(output_dir):
-            print("Removing script build dir: %s" % output_dir)
-            rmtree(output_dir)
-
-
-def make_gnulib():
-    print(">> Not a Linux system: configuring and making Gnulib libraries...")
-
-    libfilename = "%s/libgnu.a" % GNULIB_LIB_DIR
-    if os.path.isfile(libfilename):
-        print(">> Found libgnu.a... skipping configure and make")
-    else:
-        commands = ["./configure", "make"]
-        for command in commands:
-            print(">> %s" % command)
-            try:
-                check_call(command, cwd=GNULIB_BUILD_DIR)
-            except CalledProcessError:
-                raise InstallationError("Error compiling Gnulib")
-
-    if not os.path.isfile(libfilename):
-        raise InstallationError("Expected to find: %s" % libfilename)
-
-    to_rm = ["%s/getopt.h" % GNULIB_LIB_DIR]
-    for filename in to_rm:
-        if os.path.isfile(filename):
-            print(">> Removing: %s" % filename)
-            os.remove(filename)
-
-    print(">> Gnulib libraries successfully created!")
 
 if __name__ == "__main__":
-    # Configure and make gnulib if not on Linux
-    if include_gnulib:
-        try:
-            make_gnulib()
-        except InstallationError as e:
-            print(">> ERROR: %s" % e, file=sys.stderr)
-            sys.exit(1)
-
     setup(name=name,
           use_scm_version=True,
```

Comment:
Is this still used?

Reply:
Yes it's necessary for versioning information